setup-tdx-guest.sh

```diff
@@ -47,13 +47,45 @@ fi
 
 source ${SCRIPT_DIR}/setup-tdx-common
 
+usage()
```

Comment:
Could we replace the adding of an argument to this script by:
- 1) letting user modify the config file (modify `TDX_PPA` variable) to switch to dev mode.
- 2) providing a custom config file for dev mode and allow the script to accept a custom configuration file

Personally, i would go for option 1; the effort to modify `TDX_PPA` is not big and that can leave some freedom to developer to add more PPAs

What do you think ?